app/views/service_calendars/master_calendar/pppv/_pppv_calendar.html.haml

```diff
@@ -34,12 +34,12 @@
   = hidden_field_tag :service_request_id, service_request.id
   %table.table.table-bordered
     %thead
```

Comment:
Added an extra folder inside PPPV for each of the tabs for tab-specific partials